tests/components/top_bar.test.ts

```diff
@@ -873,3 +873,25 @@ test("The menu items are orderer by their sequence", async () => {
   expect(menuItems[1].dataset.name).toBe("second");
   expect(menuItems[2].dataset.name).toBe("third");
 });
+
+test.each([
+  [{ verticalAlign: "top" }, "align-top", "Vertical align"],
+  [{ verticalAlign: "middle" }, "align-middle", "Vertical align"],
+  [{ verticalAlign: "bottom" }, "align-bottom", "Vertical align"],
+  [{ align: "left" }, "align-left", "Horizontal align"],
+  [{ align: "center" }, "align-center", "Horizontal align"],
+  [{ align: "right" }, "align-right", "Horizontal align"],
+  [{ wrapping: "clip" }, "wrapping-clip", "Wrapping"],
+  [{ wrapping: "wrap" }, "wrapping-wrap", "Wrapping"],
+  [{ wrapping: "overflow" }, "wrapping-overflow", "Wrapping"],
+])(
+  "Icon in top bar matches the selected cell style",
+  async (style, expectedTitleActive, buttonTitle) => {
+    const model = new Model();
+    setStyle(model, "A1", style as Style);
+    await mountParent(model);
```

Comment:
wrap this test in a describe where you instantiate a single parent then in the test.each, set the style, await a next tick